### PR TITLE
Add MutableRealmIntegers to the CHANGELOG.

### PR DESCRIPTION
**Pull request by bmeike**
CHANGELOG update for MutableRealmIntegers

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,6 +4,7 @@
 
 * [ObjectServer] Added `SyncSession.uploadAllLocalChanges()`.
 * Added `Nullable` annotation to methods that may return `null` in order to improve Kotlin usability. This also introduced a dependency to `com.google.code.findbugs:jsr305`.
+* Added support for new data type `MutableRealmIntegers`.  The new type functions almost exactly as a reference to a Long (mutable nullable, etc) but supports `increment` and `decrement` methods, which implement a Conflict Free Replicated Data Type, whose value will converge even when changed across distributed devices with poor connections.
```
Comment (zaki50) [inline CHANGELOG.md]: We should have a reference to related issue `(#4266)` at the end of line.

**Comment by zaki50** [comment]
#5034 has been merged

File: CHANGELOG.md (hunk 1 above)
Comment (dalinaum) [inline CHANGELOG.md]: There is two spaces after `Added support for new data type MutableRealmIntegers.`

Comment (bmeike) [inline CHANGELOG.md]: Fixed.

Comment (bmeike) [inline CHANGELOG.md]: Fixed.

**Comment by bmeike** [comment]
Replaced word "functions" with the word "behaves" at the suggestion of @nhachicha

**Pull request merged**